app/elastic_search_indexes/search_index.rb

```diff
@@ -32,4 +32,8 @@ def serialize_record(record)
   def goods_nomenclature?
     false
   end
+
+  def include_in_search?
```

Comment:
from the name, seems that `include_in_search?  ` accepts an arg

what about `is_included_in_search?`